src/frisby.js

```diff
@@ -133,11 +133,12 @@ module.exports = {
   baseUrl,
   create,
   del,
+  delete: del,
   fetch,
   FrisbySpec,
   formData,
   fromJSON,
-  get: get,
```

Comment:
Some compilation steps have issues with this for some reason. Please live it as `get: get`. I know it looks stupid and redundant, but it seems to have parsing issues otherwise for ES6 -> ES5 equivalent.

Reply:
good review thanks